docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,198 @@
 ---
-title: "Getting started with Dagster Cloud | Dagster Docs"
+title: Getting started with Dagster Cloud | Dagster Cloud
 
-noindex: True
+platform_type: "cloud"
 ---
 
-# Getting started with Dagster Cloud
+# Getting Started with Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud. We'll walk you through setting up your Dagster Cloud account, configuring deployment, loading your code, launching your first run, and configuring continuous integration.
```

Comment:
"configuring deployment" reads a little odd to me, this is referring to the dagster cloud Deployment concept right? is it configuring *your* deploymment?

Reply:
It refers to hybrid versus serverless, yes.

Perhaps something about "defining" or "selecting" deployment instead?